### ISSUE TITLE
Add aria-label to header nav element

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#4302]
### Overview

The ARIA specification states that if a page includes more than one navigation landmark, each should have a unique label.  In this issue we will label the header navigation element to identify it as the primary navigation and distinguish it from the footer navigation element.  

### Action Items

- [x] The html for the header navigation can be found in _includes/main-nav.html
- [x] Change `<nav class="header-nav" id="headerNav">`
  to:
  `<nav class="header-nav" id="headerNav" aria-label="Primary">`

### Resources/Instructions
- [Nav Include](https://github.com/hackforla/website/blob/gh-pages/_includes/main-nav.html)
- [ARIA Authoring Practices Guide - Landmark Regions](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/)
- ARIA 1.2 Specification: [navigation landmark](https://www.w3.org/TR/wai-aria-1.2/#navigation)

**Comment by mademarc** [comment]
Availability: 4/5/2023  
ETA:  4/7/2023

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe I agree, and am going to prioritize this issue. 
-  I have also made an ER for a decision record. #4368

**Comment by amath95** [comment]
Availability: 04/06-04/09
ETA: EOD
